docs/modules/ROOT/pages/how-tos/testing/integration/adding.adoc

```diff
@@ -2,6 +2,9 @@
 
 In {ProductName}, you can add integration tests to verify that the individual components of your application integrate correctly, forming a complete and functional application. {ProductName} runs these integration tests on the container images of components before their release.
 
++
+NOTE: The running of an integration test run is dependant on a valid ImageIndex and pullspec, the results of a successful build pipeline. In the case of multi-arch builds, an OCI index that maps to the relevant artifacts is used.
```

Comment:
Can this be more specific to mention the exact pipelineRun results? Alternatively it can point to [the section in the architecture page](https://github.com/konflux-ci/architecture/blob/main/architecture/integration-service.md#image-extraction-details) where these are listed.